src/test/resources/application.conf

```diff
@@ -5,6 +5,7 @@ url {
 client {
     id = "aWQ=" # id
     secret = "c2VjcmV0" # secret
+    secret_path = "/a/secret/path"
```

Comment:
Should this be hashed?

Reply:
No because we've stopped encrypting things that aren't secret and the secret path isn't secret (despite its name) because it's in terraform so it's just in as plain text.